### PR TITLE
[TR] PIM-4581 : display error when completeness is inconsistent

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                    | A
| -------------------- | ---
| Bug fix?             |y
| New feature?         |n
| BC breaks?           |n
| CI currently passes? |y
| Tests pass?          |y
| Scenarios pass?      |y
| Checkstyle issues?*  |n
| PMD issues?**        |n
| Changelog updated?   |n
| Fixed tickets        |PIM-4581
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html [hunk 1]
```diff
@@ -29,6 +29,9 @@
                                     <% }) %>
                                 </span>
                                 <% } %>
+                                <% if (completeness.completeness.missing === 0 && completeness.missing.length > 0) { %>
```
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html]: yoda style pls

**Comment by solivier** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html [hunk 2]
```diff
@@ -29,6 +29,9 @@
...
+                                <span class="label label-important">Completeness error</span>
```
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html]: Add a translation key

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html (hunk 1 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html]: :+1:

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html (hunk 2 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html]: :+1:

**Comment by grena** [comment]
Small petouille then GTM

**Comment by willy-ahva** [comment]
Yoda style then GTM, translation key should never happen

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html [hunk 3]
```diff
@@ -29,6 +29,9 @@
                                     <% }) %>
                                 </span>
                                 <% } %>
+                                <% if (0 === completeness.completeness.missing && completeness.missing.length > 0) { %>
+                                <span class="label label-important">Completeness error</span>
```
Comment (jjanvier) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html]: silly question

why it's not translated ?

Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/panel/completeness.html]: yep GTM if answer and fix on this missing translation

**Pull request merged**